test/lifecycle_module_test.dart

```diff
@@ -281,15 +299,22 @@ void main() {
   Logger.root.level = Level.ALL;
   final StateError testError = new StateError('You should have expected this');
 
+  final TestTracer tracer = new TestTracer();
+  initGlobalTracer(tracer);
+  assert(globalTracer() == tracer);
```

Comment:
Same comments as in the last PR around testing and whether a non-null globalTracer is required